packages/typespec-python/scripts/eng/regenerate.ts

```diff
@@ -131,6 +131,15 @@ const EMITTER_OPTIONS: Record<string, Record<string, string> | Record<string, st
         "package-mode": "azure-dataplane",
         "package-pprint-name": "ResiliencySrvDriven2",
     },
+    "authentication/api-key": [
+        {
+        },
+        {
+            "package-name": "authentication-api-key-with-subdir",
```

Comment:
We need to add namespace with different config since there is already a package with namespace "authentication.apikey": https://github.com/Azure/autorest.python/blob/b01aea83f0941627156a37e6739730294eefad68/packages/typespec-python/test/generic_mock_api_tests/asynctests/test_authentication_async.py#L7